doc/source/Linux.rst

```diff
@@ -0,0 +1,194 @@
+Linux Tutorial
+==============
+
+This guide will give you a brief overview of how volatility3 works as well as a demonstration of several of the plugins available in the suite of plugins.
+
+Acquiring memory
+----------------
+
+Volatility3 does not provide the ability to acquire memory. In this tutorial we will see how we can use  `LiME <https://github.com/504ensicslabs/lime>`_ for this purpose. 
```

Comment:
Again, as with windows, it might be worth mentioning AVML as an alternative.

Reply:
AVML is for Linux - Acquire Volatile Memory for Linux

Reply:
It is, you're right, this is under the linux section and in comparison to LiME the LInux Memory Extractor.

Reply:
I meant in comparison to windows where we're going to mention both winpmem and also another image (such as FTK imager or similar).